dymos/trajectory/trajectory.py

```diff
@@ -1002,6 +1007,39 @@ def get_final_tbounds(phase_name, node_data, old_tmin=-INF_BOUND, old_tmax=INF_B
             err_lines = '\n'.join(errs) if len(errs) > 1 else errs[0]
             raise RuntimeError(f"{self.msginfo}: {err_lines}")
 
+    def _configure_solvers(self):
+        """
+        Automatically configure solvers for the Trajectory.
+
+        If operating under MPI and phases are connected, then
+        the default nonlinear solver will be a NonlinearBlockJac,
+        while the default linear solver will be PETScKrylov.
+
+        These solvers can be changed through the
+        'default_nonlinear_solver' and 'default_linear_solver' options.
+        """
+        if self._has_connected_phases and MPI and self.comm.size > 1:
```

Comment:
Just for clarification, why are the solvers only needed under MPI for the connected phases?

Reply:
Parallel phases are run simultaneously. If one is directly connected to another, then NLBlockJac is necessary to make sure the inputs and outputs are all in agreement

Reply:
Wait, there is an actual data connection between subsystems in a parallel group? I guess I would have to think about that.

Reply:
It's a less used capability to link phases by connection instead of constraint.  If you do that under MPI then a solver is necessary